modules/monitoring/application/controllers/ListController.php

```diff
@@ -361,13 +361,16 @@ public function eventgridAction()
         );
         $orientationBox->applyRequest($this->getRequest());
 */
+        $objectType = $this->getParam('objecttype', 'services');
```

Comment:
```suggestion
        $objectType = $form->getValue('objecttype');
```

I would suggest using this instead, because this way you can get the value from the form directly. (It also knows the defaults and that saves you some hussle)